src/main/java/ee/ut/dataObjects/Event.java

```diff
@@ -85,4 +88,8 @@ public Boolean getPlacedLeft() {
     public void setPlacedLeft(Boolean placedLeft) {
         this.placedLeft = placedLeft;
     }
+
+    public Boolean getPacked() { return packed; }
```

Comment:
I assume this is what the field "hidden" is supposed to cover already, so having two versions of it is unnecessary. If you think "packed" is a better name for it, which it might be, I'd suggest removing "hidden".